src/components/Sidebar/index.jsx

```diff
@@ -140,7 +140,6 @@ class Sidebar extends React.Component {
         onFocus={() => !this.isSidebarExpanded() && expandSidebar()}
         onMouseLeave={() => this.shouldSidebarCollapse() && collapseSidebar()}
         onBlur={() => this.shouldSidebarCollapse() && collapseSidebar()}
-        aria-expanded={this.isSidebarExpanded()}
```

Comment:
Was this intentionally removed? If so, may I ask why? I believe `aria-expanded` is needed for a11y purposes.

Reply:
It was, eslint told me I didn't need it here because it was implicit for nav components? (Or something, I don't remember the details). Seemed a bit weird to me. I could suppress the error instead?

Reply:
Did some additional reading. `aria-expanded` is typically used on interactive elements like a button, not a `nav`. I think eslint is correct here. 

Based on what I was reading, it seems `aria-expanded` is typically used to indicate that an interactive element like a button can be used to expand/collapse additional content. However, since the `nav` is technically always visible, either showing just the icons or the full menu item, I don't think `aria-expanded` actually makes sense in this case.

Let's keep it removed! The sidebar toggle in the header still refers to the `nav` sidebar through `aria-controls`.